hashmap/hashmap.mbt

```diff
@@ -187,8 +189,9 @@ fn grow[K : Hash + Eq, V](self : HashMap[K, V]) -> Unit {
   }
 }
 
+/// TODO: inline this function
 fn index[K : Hash, V](self : HashMap[K, V], hash : Int) -> Int {
-  hash.abs().land(self.capacity - 1)
+  hash.land(self.capacity - 1)
```

Comment:
cc @yj-qin the abs operation is not needed?

Reply:
yes, it's useless.